src/textual/messages.py

```diff
@@ -37,6 +37,18 @@ def can_replace(self, message: Message) -> bool:
         return isinstance(message, Layout)
 
 
+@rich.repr.auto
+class InvokeLater(Message, verbosity=3):
+    def __init__(
+        self, sender: MessagePump, callback: Callable[[], Awaitable[None]]
```

Comment:
I think this may be inherited from the current version, but are all callbacks awaitable? Not sure they are. Perhaps we need to define a callback type that accepts awaitables and plain old functions?